### PR TITLE
removes metal from requisitions console

### PR DESCRIPTION
**Pull request by Diegoflores31**
# About the pull request

Removes metal and plasteel from the ASRS orders console.
Adds metal and plasteel to the ASRS list.
reduces prices on non combat items.
updates old supply packs.
created a few new supply packs
Removes all restricted ammo except SG drums from ASRS

# Explain why it's good for the game

**METAL REMOVAL**
Requisitions department has become basically a metal factory over right now you are pretty much forced to buy metal and plasteel or you will likely get yelled at or even worse.
This PR aims to give req more freedom removing the metal from the console and making it come in ASRS supplies .

**ASRS Tweaking**
removed some of the crates of ASRS . with each new restricted weapon they were adding a new crate of it to the ASRS 
making it so all the new weapons have pretty much unlimited supply while the other restrictes lack of ammo very quickly.
This PR removes ASRS ammunition from non standard issue weapons. Req will have to manually buy it.

**Price reductions**
reduced the price of clothing items , toys , and other to allow for variety and shipside orders winout having great impact on the round.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Added new supply packs to ASRS
balance: Metal and Plasteel is no longer purchaseable in requisitions
balance: Added Metal and Plasteel to the ASRS pool
balance: Reduced the price of non combat crates and clothing
balance: Increases the amount of items in some crates
balance: Reworked a few crates from requisitions
/:cl:

### CODE REVIEW

**Comment by InsaneRed** [comment]
So how exactly does this give req more work since its just "a metal factory". You're taking away a jobs purpose to make it random so people...dont get yelled at?

If people are yelling at you, then do your job? And if its "harrasement" ahelp. Limiting building the fob and materials doesnt solve the fact that "req is boring" and req isnt "boring" by the way, it can be fun when you make it fun, the fun comes from the fact you have freedom to do whatever you want as long as you semi supply the fob.

just removing materials from being buyable doesnt solve the issue that req allegedly has, it just flops the game balance on its own as fob is very important and materials are already semi rare.

**Comment by Diegoflores31** [comment]
> and no, "part 1" doesnt excuse the fact that you're still removing one of the bigger things in balancing fob now being random instead of requiring req to do work

you spend your points actually suplying for the FOB now, you take their orders and you order it on the console . you can still use your money for FOB by buying sandbags and there is still alternative methods of obtaining metal like maint farming and recycling

if roundstart metal balance becomes an issue the amount that spawns shipside can be tweaked later on or after its actually tested

**Comment by InsaneRed** [comment]
So now you are making it a "send ammo factory" instead of send metal factory, this just seems like an useless pr trying to fix something thats already not broken in the way that it requires a fix like this

**Comment by Diegoflores31** [comment]
so true this is totally not a problem (170 pop btw)
image

**Comment by Releasethesea** [comment]
this aint it chief, its been a while since ive played req but this feels really un needed and out of touch, just because theres a few rounds where people dont go req doesnt mean you can just chop a main part of it out while saying its a metal factory, its always going to be a "BLANK factory" thats its job

**Comment by IsProbablyCatto** [comment]
> so true req is totally not a problem and its fun to play (170 pop btw) image

This was 2 hours into a round where I was CT for most of it, as well as an ASO awake

**Comment by Diegoflores31** [comment]
> this aint it chief, its been a while since ive played req but this feels really un needed and out of touch, just because theres a few rounds where people dont go req doesnt mean you can just chop a main part of it out while saying its a metal factory, its always going to be a "BLANK factory" thats its job

Req can now buy special ammunitions , maybe some grenades . restricted ammo , flamer fuel . 
all of those wont come for free in ASRS anymore and req will have to supply them depending on the marine demand

**Comment by Blundir** [comment]
> so true req is totally not a problem and its fun to play (170 pop btw) image

Nice cherry picking a random situation, i can screen cap opened sg slot on second hour and use it identically, i main qm and i can barely get on it if it's after the round start, usually it's occupied.

Also entire argument is extremely selfish, req is fun and pretty much enjoyable to play, it lacks new mechanics and polishing of existing ones (huds, visual bugs), and the only thing that this pr does, is making req even more random and even more boring, also metal isn't even a problem, inflation is a bitch and it counters all the cons of being a ""metal factory"".

**Comment by s5nt** [comment]
waiter waiter more support nerfs please

**Comment by Zonespace27** [comment]
Strongly against.

> Requisitions department has become basically a metal factory

What you're proposing is to turn it into an ammo factory instead.

> This PR aims to give req more freedom

I don't know what your idea of freedom is, but any kind of change to what req can buy will never alter their "freedom". This isn't regular ss13 where cargo's largely beholden to only themselves. Here, they have a duty they should be fulfilling, which is giving the marines what they ask for (metal and ammo). Generally, spending money on anything that's not being directly asked for (or assumed to be wanted, like steel + m41a ammo) is a waste, and removing steel from the equation won't give them more freedom in any sense.

> removed some of the crates of ASRS

You fail to explain why this is needed or good.

> making it so all the new weapons have pretty much unlimited supply

Ditto.

> reduced the price of clothing items , toys , and other to allow for variety and shipside orders winout having great impact on the round.

This is alright, but why is this in the same PR as the rest of these changes?
Also, list what you changed instead of making people root through your PR to see what prices you modified.

---

I _vaguely_ get where you're coming from with this PR, but this ain't it as a solution. Req spends most of their money on steel + plasteel because _it's the best thing you can get for your money_. Removing it is absolutely not the way to "solve" this, leaving the question of if this needs to be "solved" out of the debate.

**Comment by eMiiXi** [comment]
> Req can now buy special ammunitions , maybe some grenades . restricted ammo , flamer fuel . all of those wont come for free in ASRS anymore and req will have to supply them depending on the marine demand this PR as i said before also refreshes some of the dead old crates and reduces the prices on others . its good to experiment new ways of playing rather than pressing "buy metal 8 times at roundstart and then cryo"

As someone that played req quite a lot, if you wan't to bring people to play req, this ain't it. Neither baiting with a cutoff screenshot from a 2h+ long round. We can already do all the things mentioned above, but all of them are on stock, and more can be ordered if you manage your budget. What this achieves is just another big nerf to marines and instead of turning REQ into a metal factory, you turn them into an ammo factory, while the marines will be hurt by the more lacking materials tied to RNG. The main problem of REQ regarding attracting players to it, is the gameplay loop. Manning the early rush/desk service is the most fun of it imo, then its just dispense or accept order, pack on crate, push button. The only PR that tried to make it more fun and engaging was the forgotten truck PR that removes the launch pads. Was it old thwomper pr? i forgot. It was fun, and gets people more involved in managing supplies and delivering them, even perhaps moving groundside with the truck to deliver them instead of just pressing a button. Right now, REQ can be manned exclusively by 1 single person and the rest have nothing to do, not even packing ammo anymore cause removal of access from preps + added spare stock to REQ.

**Comment by Diegoflores31** [comment]
> > Requisitions department has become basically a metal factory
> 
> What you're proposing is to turn it into an ammo factory instead.

as said before there is more stuff to buy than ammo ( restricted weapons , spec ammo , mortar , grenades , reagent canisters(new), OT supplies (new) , flamer fuel , sand bags )

> > removed some of the crates of ASRS
> 
> You fail to explain why this is needed or good.

The removed crates are crates that basically either are commented out , dont work at all , or are not even supposed to be on the section they are currently . ( example emergency crate that gives you 2 gas masks for 1000$ ???)

> > making it so all the new weapons have pretty much unlimited supply
> 
Restricted having infinite ammo via ASRS is not supposed to happen . its braindead that just because some guns got added recently they get free ammo when most of them supposedly are balance around being "scarce" ?

> 
> > reduced the price of clothing items , toys , and other to allow for variety and shipside orders winout having great impact on the round.
> 
Yeah i should have made a list of the items modified tho most of them are non combat or not directly game changing for the average marine

> 
> I _vaguely_ get where you're coming from with this PR, but this ain't it as a solution. Req spends most of their money on steel + plasteel because _it's the best thing you can get for your money.
i have added some new crates and made some of the old ones a little better for your money.
also for the ones that think not being able to buy 10 stacks of metal roundstart is a giga req nerf. ASRS gives a reasonable amount of metal over time to keep FOB pretty much the same over time .

**Comment by Diegoflores31** [comment]
aside from that with this change i assure you req wont be completely broke by minute 25. because they "NEEDED" to make the FOB crate

**Comment by eMiiXi** [comment]
> aside from that with this change i assure you req wont be completely broke by minute 25. because they "NEEDED" to make the FOB crate

You make your money back after "needing" to make the FOB crate. And with this changes REQ may not be "broke" but fob will by minute 25. Engineers only have 1 stack of metal by default, if you even have many, wich on lowpop this will go terrible, and other roles that can help with engineer skills, now are unable to help. Aswell as suppling materials on important times at a round, such as recovering from a big siege and turning things around. Now.. Ehm, sorry, bad RNG with ASRS, no materials! since its off to the ASRS to randomly order them for you. Take green fuel as an example i had rounds where not a single canister of green fuel showed up.

File: code/datums/ASRS.dm [hunk 1]
```diff
@@ -66,42 +76,34 @@
...
 //===================================
 // Misc supplies
+/datum/supply_packs_asrs/metal_sheets
+	reference_package = /datum/supply_packs/flares
+	cost = ASRS_LOW_WEIGHT
+
+/datum/supply_packs_asrs/plasteel_sheets
+	reference_package = /datum/supply_packs/flares
+	cost = ASRS_MEDIUM_WEIGHT
+
```
Comment (Zonespace27) [inline code/datums/ASRS.dm:106]: `reference_package`

Comment (Diegoflores31) [reply code/datums/ASRS.dm:106]: LMAO AHHAHA

Comment (Diegoflores31) [reply code/datums/ASRS.dm:106]: will fix rn

**Comment by neeshacark** [comment]
this is a joke pr, right?

right?

...

RIGHT?

**Comment by Diegoflores31** [comment]
> You make your money back after "needing" to make the FOB crate. And with this changes REQ may not be "broke" but fob will by minute 25. Engineers only have 1 stack of metal by default, if you even have many, wich on lowpop this will go terrible, and other roles that can help with engineer skills, now are unable to help. Aswell as suppling materials on important times at a round, such as recovering from a big siege and turning things around. Now.. Ehm, sorry, bad RNG with ASRS, no materials! since its off to the ASRS to randomly order them for you. Take green fuel as an example i had rounds where not a single canister of green fuel showed up.

as mentioned before, you can still buy sandbags/portacades if you want to provide extra or emergency FOB reconstruction. 
about roundstart . i will likely add some roundstart metal reserves on the almayer depending on the map/pop. if its too little . i have yet to test how much metal you can get from ASRS in those first 25 minutes pre-deployment

**Comment by Skatuk** [comment]
If you want req to be popular rework the Foxhole funny PR instead. Simply not everyone wants to sit at requisitions who always have a small budget and when you request something like M2C, they say no because it costs too much. Lower weapon costs and ways to get money for requistions like for example on TGMC if I remember, selling xeno's corpses gives you money.

**Comment by Lukas-Seltz** [comment]
> as said before there is more stuff to buy than ammo ( restricted weapons , spec ammo , mortar , grenades , reagent canisters(new), OT supplies (new) , flamer fuel , sand bags )

Most of that can already be bought from req, I used to play a lot of OT and I will tell you it is almost never worth it to go to req for chems. You should always go to medical because what is the point of wasting 2000 points when you can just wait 10 minutes and be completely restocked. The only reason OT should need to go to req is to get metal and get other sheets of minerals which you are effectively removing. This would just not be nerfing FOBs and such you would be nerfing all roles and jobs that require metal, such as OT, Medical, Comtechs, MTs, and CE, these are just examples. If medical does not have metal, they can't give people new limbs which means you can have 20 marines sitting on the ship unable to do anything because they do not have limbs that they need metal for.

**Comment by Diegoflores31** [comment]
> Most of that can already be bought from req, I used to play a lot of OT and I will tell you it is almost never worth it to go to req for chems. You should always go to medical because what is the point of wasting 2000 points when you can just wait 10 minutes and be completely restocked. The only reason OT should need to go to req is to get metal and get other sheets of minerals which you are effectively removing. This would just not be nerfing FOBs and such you would be nerfing all roles and jobs that require metal, such as OT, Medical, Comtechs, MTs, and CE, these are just examples. If medical does not have metal, they can't give people new limbs which means you can have 20 marines sitting on the ship unable to do anything because they do not have limbs that they need metal for.

I am a plat OT , you some times need to ask req( pretty sure the price is 1000 , i will check on that )  . Since not allways medbay is cooperative or competent enought

Your second point about metal needed shipside is a very valid concern but with the metal that is available shipside in engineering is more than enough to print a full set of limbs for every marine deployed.

Req usually does not order metal for OT but rather snack some of thr FOB metal for OT . You can also farm barrels/recycle . I will see if i can maybe set a crate that come every X time or purchaseable ID locked to shipside personell

**Comment by Zonespace27** [comment]
Thank you for the PR, but this isn't something we're interested in at the moment.

**Pull request closed without merging**